native/explorer/src/series.rs

```diff
@@ -708,6 +708,13 @@ pub fn s_pow(data: ExSeries, exponent: f64) -> Result<ExSeries, ExplorerError> {
     Ok(ExSeries::new(s.pow(exponent)?))
 }
 
+#[rustler::nif]
+pub fn s_int_pow(data: ExSeries, exponent: i64) -> Result<ExSeries, ExplorerError> {
```

Comment:
You should actually be able to just use `u32` in the signature here for `exponent` and `rustler` should automatically marshall correctly.